### ISSUE TITLE
Layout element for Recurring Events

### ISSUE THREAD

**Issue by Siggik** [dukop/django-dukop#289]
Er lidt i tvivl om det er en bug eller med vilje. Jeg tænker ikke det umiddelbart er  smart. For mig bliver "All events next 30 days" listen uoverskuelig, og jeg vil mene at pointen med "recurring event" tabellen er at undgå netop det? 

Fyi. Har skrevet med @troldmand i dag om at kigge på "recurring event" listen/tabellen for at gøre den mere overskuelig at se på. Den er svær at skabe sig et overblik på som den ser ud nu.  Men måske @troldmand også kan prøve at gennemskue række følgene af listerne/modulerne på forsiden og deres navne osv.. Der er noget der ikke er lige i skabet, men har ikke helt svaret på hvordan det kan gøre bedre, udover ovenstående forslag.

**Comment by troldmand** [comment]
Here's a simple styling proposal for desktop and mobile: 
<img width="955" alt="Screenshot 2022-04-20 at 18 13 47" src="https://user-images.githubusercontent.com/9819978/164275971-2ee5708d-f3ee-40fe-b0b5-a35d0285da28.png">
<img width="285" alt="Screenshot 2022-04-20 at 18 13 58" src="https://user-images.githubusercontent.com/9819978/164275982-f1d1b149-9725-43af-bd63-f0820e257ff2.png">

@Siggik, what do you think? This is just like 5 -6 CSS properties and can be done very fast.

**Comment by Siggik** [comment]
I like the smal symbols, makes it intuitive to se what the info is referring to. Could there be a sorting for every weekday, like the old site. I think it would make it faster to get an overview: 
![signal-2022-04-20-185400](https://user-images.githubusercontent.com/7570153/164283367-9f4ea8f2-a35a-43f8-804a-62399591dddc.png)

**Comment by benjaoming** [comment]
Vi bruger dette ikon-sæt:

https://github.com/dukop/django-dukop/tree/main/src/dukop/static/img/remixicons
https://remixicon.com/

**Comment by troldmand** [comment]
@Siggik, det ser jo perfekt ud! 

For at lave det, skal vi vel nok hellere spytte et andet udgangspunkt ud med python, wouldn't you say, @benjaoming ? Og så kan jeg style det i CSS og sætte ikon på. 

**Så skal vi bruge den her 'model':** 

Dag
- Title
- Adresse, tidspunkt

**Hvor nu får vi:**

Title 
- Næste dag
- "weekly"+dag+tid 
- Sted

**Comment by Siggik** [comment]
ja, gir mening. Men den der "weekly", gir stadig mening. Kan også være hver anden uge. hmm, men det kan faktisk også være den første i måneden, den sidste i måneden osv., så falder det uden for kategori. Hmm, det er sku ikke nemt. Jeg tror løsningen er nu, at de gentagende events ikke blir vist de uger, hvor de ikke er aktive, det er heller ikkke den bedste løsning.

**Comment by Siggik** [comment]
Så har jeg været i tænke pause. Dette er bud på den mindst ringe løsning. Vi viser alle de gentagende events også selv om de kun sker en gang om måneden. Og viser dem på samme måde som i den store liste, så folk øjne ikke skal lurer to systemer:

Så noget al  den her model. Kan ikke helt gennemskue hvad vil se bedst ud:

Dag

    Tidspunkt, Hver uge( eller i ulige uge/ i lige uger, først uge i måneden etc.), Title.  Adresse, 

_eller_

Dag

    Tidspunkt, Title. Hver uge( eller i ulige uge/ i lige uger, først uge i måneden etc.),  Adresse, 

Fremfor:

Title

    Næste dag
    "weekly"+dag+tid
    Sted

**Comment by benjaoming** [comment]
Jeg har opdateret nuværende visning.

Jeg tænkte på, om ikke bare jeg skal merge'e #318 og så kan du arbejde videre ud fra dette @troldmand ?

**Comment by troldmand** [comment]
Amazing, I will get to work later :)

**Comment by benjaoming** [comment]
Oops, this was closed by mistake

**Comment by troldmand** [comment]
![Uploading Screenshot 2022-04-26 at 21.01.14.png…]()

**Comment by troldmand** [comment]
@benjaoming , kan du give mig et fif til at få oprettet test-data til recurring events, så jeg kan style tabellen? 

Jeg har forsøgt `python3 manage.py calendar_fixtures` fra `readme`, men der kommer ingen recurring events :/ 

![Screenshot 2022-04-26 at 21 02 23](https://user-images.githubusercontent.com/9819978/165373285-4c04c694-dd1c-4669-8a97-95879228b075.png)

**Comment by benjaoming** [comment]
Ååååh det er desværre manuelt :/

**Comment by benjaoming** [comment]
Det er lokalt på hans computer - skal oprettes manuelt (er heldigvis ikke så svært), indtil at vi kan auto-generere noget test-data.

**Comment by Siggik** [comment]
Fyi. Der er kommet en masse faste begivenheder fra Ungdomshuset nu. Folkets oprettet også deres de næste dage. Måske det gør det lidt nemmere at se hvordan det er bedst at stille det op, nu hvor der mange faste events.

**Comment by Siggik** [comment]
Forslag til små simple ændringer, som gør det hele mere overskueligt. 

![gentagende events layout](https://user-images.githubusercontent.com/7570153/170668155-26276eab-0c4d-4346-b9d8-c16d54b502de.png)

**Comment by benjaoming** [comment]
Vi er rimelig langt med dette, men vi kan nok kunne finde på nogle forbedringer til det nuværende element.

* Vis pr. ugedag i stedet for pr. dato
* Visning af næste forekomst + interval

F.eks.

**Wednesday**
Baraften, Pseudobar, Weekly - next time July 5th
Drum circle, Another galaxy, Odd weeks - next time July 12th
